doc/manual/modules/performance/proc_setting-the-disk-scheduler-using-tuned.adoc

```diff
@@ -57,39 +67,51 @@ $ tuned-adm active
 # mkdir /etc/tuned/[replaceable]__my-profile__
 ----
 
-. Find the World Wide Name (WWN) identifier of the selected block device:
+. Find the system unique identifier of the selected block device:
 +
 [subs=+quotes]
 ----
-$ udevadm info --query=property --name=/dev/[replaceable]__device__ | grep WWN=
+$ udevadm info --query=property --name=/dev/_device_ | grep -E '(WWN|SERIAL)'
```

Comment:
Hi Jiří. So the `[replaceable]` tag is actually largely a remnant from the RHEL7 docs that were written in docbook instead of Asciidoc and docbook automatically created these tags. Going through some past meetings, I was actually incorrect to delete the tags vis-a-vis our docs standards.  From our March 3rd docs most-hands mtg:
"[H]ere’s the recommended list of semantic tags (formerly DocBook tags) to be used as Asciidoc attributes for a bigger value of the source code and future processing:
**application, citetitle, command, filename, function, guilabel, guimenu, guisubmenu, literal, package, option, parameter, replaceable, systemitem, varname**"

I can go through and re-add at least the `[replaceable]` tags before this PR gets accepted, if you think that's a good route to go.

I agree with the proliferation of RHEL-specific content and actually have a ticket to begin working on this after we get through the workloads we currently have slated in docs. Largely a lot of `ifdef`s, like you mentioned :D If you like, I can tag you in the Jira we are using to track this.

Reply:
Hi Jacob,
it was actually unclear to me what `[replaceable]` was used to as I was writing docs for `TuneD` myself and was hoping that you might shed some light on that.  To answer the question if we need to re-add the `[replaceable]s`, I'd need to understand the value and I still do not.  Do you have any links to docs what this tag means exactly?  Thanks!

Reply:
Hi Jiří. I've managed to find this doc which gives guidance on how to use the tags in AsciiDoc (it's generally about migrating from docbook but this should give context on when to use the given attribute tag. I'm actually hearing conflicting things on whether or not we still use these across CCS, but I'll have an answer today/soon): 
http://jenkinscat.gsslab.pnq.redhat.com:8080/job/Migrating-RHEL-Docs-from-DocBook-to-AsciiDoc/lastSuccessfulBuild/artifact/tmp/en-US/html-single/index.html#docbook-versus-asciidoc-markup-reference